modules/home/api.php

```diff
@@ -1,65 +1,19 @@
 <?php
 namespace Elementor\Modules\Home;
 
+use Elementor\Data\EditorAssets\EditorAssetsAPI;
 use Elementor\Modules\Home\Classes\Transformations_Manager;
 
-class API {
```

Comment:
I have extracted this code to a general class, in order make it reusable for other modules.